### PR TITLE
Obg 1014/fixe warning after angular update

### PR DESCRIPTION
**Pull request by collins-k**

### CODE REVIEW

**Comment by valb3r** [comment]
There is an excessive amount of trailing commas, can we fix that?

**Comment by collins-k** [comment]
I will take a look on our prettier config if we can fix it

File: fintech-examples/fintech-ui/src/app/services/auth.service.spec.ts [hunk 1]
```diff
@@ -66,6 +74,6 @@ describe('AuthService', () => {
     authService.logout();
 
     // user must be navigated to login page
-    expect(navigateSpy).toHaveBeenCalledWith(['/login']);
+    expect(navigateSpy).toHaveBeenCalledWith(['/'.concat(RoutingPath.LOGIN)]);
```
[the remark this answers is not in the document]
Comment (collins-k) [reply fintech-examples/fintech-ui/src/app/services/auth.service.spec.ts:77]: Yep. It's even better

**Comment by collins-k** [comment]
> There is an excessive amount of trailing commas, can we fix that?

I fixed it

**Review by valb3r — APPROVED** [review]

File: consent-ui/src/app/ais/entry-page/initiation/common/dedicated-access/dedicated-access.component.ts [hunk 2]
```diff
@@ -51,9 +51,9 @@ export class DedicatedAccessComponent implements OnInit {
     consentObj.consent.access.availableAccounts = null;
     consentObj.consent.access.allPsd2 = null;
 
-    consentObj.consent.access.accounts = this.accounts.map(it => it.iban);
-    consentObj.consent.access.balances = this.accounts.map(it => it.iban);
-    consentObj.consent.access.transactions = this.accounts.map(it => it.iban);
+    consentObj.consent.access.accounts = this.accounts.map((it) => it.iban);
+    consentObj.consent.access.balances = this.accounts.map((it) => it.iban);
+    consentObj.consent.access.transactions = this.accounts.map((it) => it.iban);
```
Comment (max402) [inline consent-ui/src/app/ais/entry-page/initiation/common/dedicated-access/dedicated-access.component.ts:56]: also it adds unneeded parentheses

Comment (collins-k) [reply consent-ui/src/app/ais/entry-page/initiation/common/dedicated-access/dedicated-access.component.ts:56]: > also it adds unneeded parentheses

Parentheses are needed for the readability of the code in the IDE. For the automatic display of the object type within the parentheses

Comment (max402) [reply consent-ui/src/app/ais/entry-page/initiation/common/dedicated-access/dedicated-access.component.ts:56]: then ok. I just noticed that default behaviour of prettier changed (arrow-parens default value changed from avoid to always in v2.0.0)

**Review by max402 — APPROVED** [review]

**Pull request merged**